src/pages/home/report/ReportActionsList.js

```diff
@@ -17,11 +17,16 @@ import participantPropTypes from '../../../components/participantPropTypes';
 import * as ReportActionsUtils from '../../../libs/ReportActionsUtils';
 import reportActionPropTypes from './reportActionPropTypes';
 import CONST from '../../../CONST';
-import * as StyleUtils from '../../../styles/StyleUtils';
 import reportPropTypes from '../../reportPropTypes';
 import networkPropTypes from '../../../components/networkPropTypes';
 import withLocalize from '../../../components/withLocalize';
 
+// This is a workaround to a reanimated issue -> https://github.com/software-mansion/react-native-reanimated/issues/3355
+if (process.browser) {
+    // eslint-disable-next-line no-underscore-dangle
+    window._frameTimestamp = null;
+}
```

Comment:
It looks like https://github.com/software-mansion/react-native-reanimated/issues/3355 was fixed in https://github.com/software-mansion/react-native-reanimated/commit/2fd0da8818b63727a311eeb45817d949021d37b9 which was released in 3.0.0, however you're still using 3.0.0-rc.10. It's okay to keep this workaround as for now but once you upgrade to 3.0.0+ (currently the latest version is 3.0.2) you should be safe to get rid of it.

Reply:
Hey there :-) 
Using reanimated@3.0.2 I still seem to have this problem with next.js unfortunately.
next.js version is 12.1.5